src/github/index.ts

```diff
@@ -1,8 +1,16 @@
+import enterpriseCompatibility from '@octokit/plugin-enterprise-compatibility'
+import retryPlugin from '@octokit/plugin-retry'
+import throttlePlugin from '@octokit/plugin-throttling'
 import Octokit from '@octokit/rest'
+
 import { addGraphQL } from './graphql'
 import { addLogging, Logger } from './logging'
 import { addPagination } from './pagination'
-import { addRateLimiting } from './rate-limiting'
+
+const ProbotOctokit = Octokit
+  .plugin(throttlePlugin)
+  .plugin(retryPlugin)
```

Comment:
super stoked about these two :) /cc @SGrondin

Reply:
Woohoo!!! 🎉 This is so much better than the custom throttling that existed in probot. What I'm even more excited about is to add both to octokit/rest directly. I've been waiting for that for **years**.